src/shared/components/search-result/search-result.component.html

```diff
@@ -4,6 +4,10 @@ <h2 class="heading-h2">{{ caseState ? 'Case List' : 'Search result' }}</h2>
     <div *ngIf="(hasResults() || hasDrafts())" class="pagination-top"
          attr.aria-label="{{ getTotalResults() }} results have been found">
       <span class="text-16">Displaying {{ getFirstResult() }} - {{ getLastResult() }} out of {{ getTotalResults() }} results</span>
+      <span class="cline-any cline-neutral">&nbsp;</span>
+      <span class="cline-any cline-neutral">&nbsp;</span>
+      <span class="cline-any cline-neutral">&nbsp;</span>
+      <span><a class="govuk-link" href="javascript:void()" (click)="clearSelection()">Reset case selection</a></span>
```

Comment:
Why are there 3 spaces here? Should this not be achieved by applying CSS to `<span><a class="govuk-link"...`?